crates/core_arch/src/acle/mod.rs

```diff
@@ -119,6 +119,9 @@ mod simd32;
 ))]
 pub use self::simd32::*;
 
+#[cfg(any(target_arch = "aarch64", target_feature = "v7"))]
+pub mod crc;
```

Comment:
Add a `pub use` here to re-export the contents of the module and make the module private.